loomi/regulariza-diagonal-client
# Feature/integration login

##  Login Integration

---

**What I did:**

- Integrate login
  - user data is still mocked

**How to test:**

1. In your browser go to [stage](http://project.netlify.app/) or [localhost](http://localhost:3000/)
2. Login with the credentials of Alyson and see if everything's doing fine
3. Test the things that you think are worth testing, even the ones that are not in this description
